BrewMesModulesParent/api/src/main/java/com/brewmes/api/BatchController.java

```diff
@@ -90,4 +91,22 @@
             return new ResponseEntity<>("Error: File not found", HttpStatus.NOT_FOUND);
         }
     }
+
+    /**
+     * Gets {@code variables}: {@code id}, {@code type}, {@code amount} and {@code speed} from the latest {@code batch} and displays them in {@codeJsonObject} format
+     * @param id is the ID of the connection
+     * @return {@code 200 OK} and a {@code JsonObject} with the information if successful; otherwise {@code 404 NOT FOUND}
+     */
+    @GetMapping(value = "/batch-of-machine/{id}")
+    public ResponseEntity<Object> getStaticBatchVariablesForMachine(@PathVariable("id") String id) {
+        JsonObject jsonObject = getter.getStaticBatchVariables(id);
```

Comment:
Should this return a Batch object instead?

Reply:
A Batch holds loads of irrelevant information especially machinedata list.

Reply:
Ofc, but is there a reason for it not be Batch object with null on the fields that we do not need in this aspect? I don't see the reasoning to make a JsonObject instead of a new Batch object.

Reply:
I see the reason baby <3